dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/BeamSqlFnExecutor.java

```diff
@@ -278,6 +279,12 @@ static BeamSqlExpression buildExpression(RexNode rexNode) {
         case "SIGN":
           ret = new BeamSqlSignExpression(subExps);
           break;
+        case "POWER":
```

Comment:
I didn't found the `pow`(just find `power`) in calcite reference or calcite source code, can you point me where is `pow` defined? @app-tarush